catalog/ui/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -51,6 +52,7 @@ module.exports = Marionette.LayoutView.extend({
         this.$el.toggleClass('is-text', false);
         this.$el.toggleClass('is-basic', false);
         this.$el.toggleClass('is-advanced', false);
+        this.$el.toggleClass('is-custom', false);
```

Comment:
I don't think we need to toggle these classes (`is-text`, etc.), nothing appears to utilize them.

Reply:
Removed